pulp_container/app/tasks/sign.py

```diff
@@ -46,10 +47,21 @@ def sign(repository_pk, signing_service_pk, reference, tags_list=None):
     latest_version = repository.latest_version()
     if tags_list:
         latest_repo_content_tags = latest_version.content.filter(
-            pulp_type=Tag.get_pulp_type(), pk__in=tags_list
+            pulp_type=Tag.get_pulp_type(),
+            pk__in=tags_list,
+        ).exclude(
+            Q(container_tag__endswith=".sig")
+            | Q(container_tag__endswith=".att")
+            | Q(container_tag__endswith=".sbom")
```

Comment:
I wonder if there are any more types we would like to ignore. I could not find any while briefly googling the subject.

Reply:
instead of traversing into another table do it directly from the Tag.name on line 65